src/os/bluestore/BlueStore.cc

```diff
@@ -10305,8 +10305,6 @@ int BlueStore::_do_write(
   o->extent_map.compress_extent_map(dirty_start, dirty_end - dirty_start);
   o->extent_map.dirty_range(dirty_start, dirty_end - dirty_start);
 
-  r = 0;
-
  out:
```

Comment:
Thanks for patience. I scrutinize _do_write_alloc and _do_gc, and I think they will not return a positive integer. At the same time, add this does not affect the function..